### DISCUSSION TITLE
Reply to #362

### DISCUSSION THREAD

**Discussion by garry-ut99** [BurntSushi/ripgrep#2867] [General] [answered]
A reply to https://github.com/BurntSushi/ripgrep/issues/362#issuecomment-2273392698

> BurntSushi : I'll repeat one last time: please open a new issue with a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example). 

No, I'm not interested anymore in contributing to this repository, nor reporting any bugs/issues to it, nor I care about anything being fixed here, find other people who will report bugs/issues for you. 

You're too aggressive/rude, you agressively locked the thread, providing no answer for any questions, nor giving any chance to reply, forcing me to create a new Discussion just to reply to you. I'm not interested in opening new issues when no reason is provided, just because you want me to do so, nor I'm a dog for you orders (maybe in your dreams only). If you don't want to cooperate politely then neither do I.

> BurntSushi : If you want my help,

In your dreams, I never ever wanted your's or anybody's help, I just reported a bug/issue to notify people about such issue.  It will be fixed, fine, it won't, I don't give a s.., the world doesn't end for me, I don't demand anything from anyone, except some respect, which you lacks.

 > BurntSushi : that's what I require.

What I require is a bit of respect, not rudely locking threads, but I guess you locked the thread, because you run out of arguments pretty fast, you can m4sturb4t3 your "lock thread" or any other buttons, with other people, not with me.

**Comment by BurntSushi** [comment, marked as answer]
For context, these were my "aggressive/rude" comments:

> @garry-ut99 Please open a new issue and provide an MRE. I can't help you if you can't provide a way for me to reproduce the behavior you're seeing. You don't even share the actual rg you're using!

And:

> I'll repeat one last time: please open a new issue with a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example). If you want my help, that's what I require.

But okay, play stupid games, win stupid prizes.

\*plonk\*

(Apparently you've already been [banned from GitHub before](https://github.com/OldUnreal/UnrealTournamentPatches/issues/114).)
